frontend/src/router.ts

```diff
@@ -4,6 +4,7 @@ import ArticleView from "@/views/ArticleView.vue";
 import ClientListView from "@/views/ClientListView.vue";
 import ClientView from "@/views/ClientView.vue";
 import DocumentView from "@/views/DocumentView.vue";
+import OrderListView from "@/views/OrderListView.vue";
 
 export const ROUTES = {
   CLIENTS: { name: "clients", path: "/clients", label: "Kunden" },
```

Comment:
The "CLIENT" and "CLIENTS" routes have a separate URL for now.

Do we want to keep that? Personally, I think having one route with an optional ID makes more sense (and aligns with typical backend designs).

I.e. something like this


```txt
clientList -> the-url.com/client
singleClient -> the-url.com/client/:client-id
```


What do you think?